pkg/webhook/policy.go

```diff
@@ -174,6 +174,15 @@ func (h *validationHandler) getDenyMessages(res []*rtypes.Result, req admission.
 	var msgs []string
 	for _, r := range res {
 		if r.EnforcementAction == "deny" || r.EnforcementAction == "dryrun" {
+			resourceName := req.AdmissionRequest.Name
```

Comment:
Do we want to move this inside of `if *logDenies` to avoid spending time deserializing an object we aren't going to reference?